src/jetstream/passthrough.go

```diff
@@ -398,6 +398,24 @@ func (p *portalProxy) DoProxySingleRequest(cnsiGUID, userGUID, method, requestUr
 	return responses[req.ResultGUID], err
 }
 
+// Convenience helper for a single request using a token
+func (p *portalProxy) DoProxySingleRequestWithToken(cnsiGUID string, token *interfaces.TokenRecord, method, requestURL string, headers http.Header, body []byte) (*interfaces.CNSIRequest, error) {
```

Comment:
Question - will not have
cnsiRequest.UnescapePath = true // not needed as only used for projects?
cnsiRequest.AllowNoToken = true // not needed as only used for requests without a user/token